src/findings.ts

```diff
@@ -0,0 +1,23 @@
+import { Finding, TransactionEvent, FindingSeverity, FindingType } from "forta-agent";
+
+export const createFinding = (
+  name: string,
+  description: string,
+  alertId: string,
+  txEvent: TransactionEvent,
+  data: string
```

Comment:
Instead of passing the "data", it would like nicer if you passed the actual arguments of the function called, i.e. `agentId`, `metadata`, `chainIds`

Those can be extracted from the `agentDeploymentEvents` (or `agentDeploymentCalls` if you change that)